### PR TITLE
Add a resources page

### PR DESCRIPTION
**Pull request by MarkBennett**
This is a placeholder and not ready to link from the main page, it is ready to share with sponsors who want to add offers and is ready to accept PRs to add resources.

Once it's complete enough we should add it to the site menu and start sharing with teams.

Note, I used some of resources in the `elements.html` page as templates for these changes.

### CODE REVIEW

File: resources.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+<html>
+  <head>
+    <title>CODEVID-19 - Resources</title>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, user-scalable=no"
+    />
+    <link rel="stylesheet" href="assets/css/main.css" />
+    <noscript><link rel="stylesheet" href="assets/css/noscript.css"/></noscript>
```
Comment (ajyong) [inline resources.html:16]: Take a look at `index.html`, I think Google Analytics should be added here as well

File: resources.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    </header>
+
+    <!-- Wrapper -->
+    <div id="wrapper">
+      <!-- Main -->
+      <section id="main" class="wrapper">
+        <div class="inner">
+          <h1 class="major">Resources & Offers</h1>
+          <ul class="actions stacked">
+            <li><a href="https://github.com/devedmonton/codevid19.com/edit/master/resources.html" class="button primary fit">EDIT THIS PAGE ON GITHUB TO ADD NEW RESOURCES AND OFFERS</a></li>
```
Comment (ajyong) [inline resources.html:51]: I think the button styles make it all-caps without needing to type it in all-caps, I could be wrong though.

File: resources.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+          </ul>
+        </div>
+      </section>
+    </div>
+
+    <!-- Footer -->
+    <footer id="footer" class="wrapper alt">
+      <div class="inner">
+        <ul class="menu">
+          <li>&copy; Untitled. All rights reserved.</li>
```
Comment (ajyong) [inline resources.html]: Footer needs to be updated, look at `index.html`

**Review by ajyong — CHANGES_REQUESTED** [review]
Still some boilerplate that needs to be modified, otherwise LGTM

EDIT: Just re-read your comment
> This is a placeholder and not ready to link from the main page

Ah.  I guess my feedback on boilerplate wasn't helpful, except for some of the basics like adding Google Analytics and updating the footer?

File: resources.html [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    <!-- Wrapper -->
+    <div id="wrapper">
+      <!-- Main -->
+      <section id="main" class="wrapper">
+        <div class="inner">
+          <h1 class="major">Resources & Offers</h1>
+          <ul class="actions stacked">
+            <li><a href="https://github.com/devedmonton/codevid19.com/edit/master/resources.html" class="button primary fit">EDIT THIS PAGE ON GITHUB TO ADD NEW RESOURCES AND OFFERS</a></li>
+          </ul>
+          <span class="image fit"><img src="images/pic04.jpg" alt=""/></span>
```
Comment (MandyMeindersma) [inline resources.html:53]: not quite sure what this image is?

**Comment by MandyMeindersma** [comment]
other than the image it looks good to me

File: resources.html (hunk 4 above)
Comment (ajyong) [reply resources.html:53]: Oh, think this is boilerplate as well.  Pic04 was a blurry example image, iirc

File: resources.html (hunk 1 above)
Comment (MarkBennett) [reply resources.html:16]: Done.

File: resources.html (hunk 3 above)
Comment (MarkBennett) [reply resources.html]: Good catch. Thanks!

File: resources.html (hunk 4 above)
Comment (MarkBennett) [reply resources.html:53]: Yes. Ideas on what to replace it?

**Comment by MarkBennett** [comment]
Going to merge so this is available for others to hack on. I addressed all comments but the image.

**Pull request merged**